Project-I-Social-Card/social-card/src/components/FooterComponents/FooterIcons.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './Footer.css';
+
+
+const Icons = () => {
+    return (
+        <div className='icon-row'>
+            <i className='fas fa-comment'></i>
+            <div className='countable'  onClick={incrementHearts}>
+                <span id='repost-count'>{repostCount}</span>
+                <i className="fas fa-sync-alt" ></i>
+            </div>
+            <div className='countable' onClick={incrementPosts}>
+                <span id='heart-count'>{heartCount}</span>
+                <i className="fas fa-heart"></i>
+            </div>
+            <i className="fas fa-envelope"></i>
+        </div>
+    )
+}
+
+let heartCount = 0;
+let repostCount = 0;
+
```

Comment:
Nice job making these counters to increase their value on each click, but unfortunately in React this will not render the values to the DOM. You will soon learn how to manage data using `state` and make this `onClick` functionality works.